packages/block-library/src/cover/style.scss

```diff
@@ -12,40 +12,6 @@
 	align-items: center;
 	overflow: hidden;
 
-	&.has-left-content {
-		justify-content: flex-start;
-
-		h2,
-		.wp-block-cover-image-text,
-		.wp-block-cover-text {
-			margin-left: 0;
-			text-align: left;
-		}
-	}
-
-	&.has-right-content {
-		justify-content: flex-end;
-
-		h2,
-		.wp-block-cover-image-text,
-		.wp-block-cover-text {
-			margin-right: 0;
-			text-align: right;
-		}
-	}
-
-	h2,
-	.wp-block-cover-image-text,
-	.wp-block-cover-text {
-		font-size: 2em;
-		line-height: 1.25;
-		z-index: 1;
-		margin-bottom: 0;
-		max-width: $content-width;
-		padding: $block-padding;
-		text-align: center;
-	}
-
 	h2:not(.has-text-color),
```

Comment:
Do we need this here?

Reply:
Yes, we need these styles because of the previous versions, they were always white. Current versions inriht the color, I move these styles to the deprecated zone.